packages/concerto-core/lib/modelutil.js

```diff
@@ -331,6 +338,21 @@ class ModelUtil {
             `${MetaModelNamespace}.ObjectMapValueType`
         ].includes(value.$class);
     }
+
+    /**
+     * Returns the corresponding ClassDeclaration representation of the Map Type
```

Comment:
Typo in the jsdoc - this is not map specific. 

Not sure we need this function. ModelFile.getType takes a short type name and returns the FQN, taking care of primitives and imported types.

So, this could be:

```
// decl will be null for primitive types ...

const decl = !ModelUtil.isPrimitiveType((shortName) ? modelFile.getModelManager().getType(modelFile.getType(shortName)) : null;
```